litmus-portal/graphql-server/manifests/cluster/3b_agents_deployment.yaml

```diff
@@ -23,6 +22,8 @@ metadata:
 stringData:
   CLUSTER_ID: #{CLUSTER_ID}
   ACCESS_KEY: #{ACCESS_KEY}
```

Comment:
CUSTOM_TLS_CERT: "#{CUSTOM_TLS_CERT}"

Hi Axit, I have already mentioned in the issue. You need to think before you push. Also you have to change the logic as told by @neelanjan00. GD!

Reply:
Yes, I am working on it.
Since it is open-source it's OK to make mistakes.